src/clj/fluree/db/flake/transact.cljc

```diff
@@ -120,6 +121,9 @@
 (defn stage
   [db fuel-tracker context identity author annotation raw-txn parsed-txn]
   (go-try
+    (when (novelty/max-novelty? db)
+      (throw (ex-info "Maximum novelty exceeded, try again later."
+                      {:status 429 :error :db/max-novelty-exceeded})))
```

Comment:
I think the status code should be [503](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503)